tests/PHPUnit/Framework/TestCase/UnitTestCase.php

```diff
@@ -18,16 +21,45 @@
  */
 abstract class UnitTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
> Ideally do we agree that we should remove that class instead of making more tests extend it 

Ideally yes, but we're not in an ideal world. It is ok to leave this class there (and to extend it) even if it would just do nothing. I would as well not change the naming to `PiwikTestCase` or so to have a consistent naming with `UnitTestCase`, `IntegrationTestCase` and `SystemTestCase`.